app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
If you inherit this class also from the ABC class, you will get rid of the message from PyCharm like "Class Elf must implement all abstract methods". It's just a small advice. It's up to you to accept it or not.